bin/spark-evaluate.js

```diff
@@ -30,10 +30,11 @@ await migrateWithPgConfig({ connectionString: DATABASE_URL })
 
 const { ieContract, provider } = await createMeridianContract()
 
-const signer = ethers.Wallet.fromPhrase(WALLET_SEED, provider)
-const walletDelegatedAddress = newDelegatedEthAddress(/** @type {any} */(signer.address), CoinType.MAIN).toString()
+const wallet = ethers.Wallet.fromPhrase(WALLET_SEED, provider)
+const signer = new ethers.NonceManager(wallet)
+const walletDelegatedAddress = newDelegatedEthAddress(/** @type {any} */(wallet.address), CoinType.MAIN).toString()
```

Comment:
unfortunately `signer.signer.address` isn't available in types, although in praxis it is